test/test.cli.ts

```diff
@@ -29,27 +29,27 @@ describe('cli', function () {
     const res = await execa('linkinator', ['test/fixtures/basic'], {
       reject: false,
     });
-    assert.match(res.stderr, /ERROR: Detected 1 broken links/);
+    assert.ok(res.stderr.includes('ERROR: Detected 1 broken links'));
```

Comment:
I know that it seems like a lot bringing `chai` just for this ... but it's exactly why I used it instead of basic `assert` :) With the `assert.ok` approach, when this assertion fails I will just see something like `expected value TRUE` coming back from `mocha`, and have no idea which strings it was comparing.  This makes debugging the failing test ... annoying.  When you use `assert.match` chai treats things just as it would for a `assert.strictEqual`, where the tooling shows you both the left and the right side you're comparing.  If node adds an `assert.match` to core, I would drop `chai` in a second ... but this is pretty valuable and doesn't cost much.

Reply:
All good, I thought it was something that slipped :)